src/commands/generate/app.ts

```diff
@@ -14,10 +14,15 @@ import { vpAdapterService } from '../../services/affinidi/vp-adapter'
 import { createAuth0Resources } from '../../services/generator/auth0'
 import { configureAppEnvironment } from '../../services/generator/env-configurer'
 
+const APPS_INFORMATION_GITHUB_LOCATION = 'samples/auth0-nextjs-nextauthjs/tsconfig.json' // TODO: Change
```

Comment:
Change what?

Reply:
Ah yes, I had to change the url to the actual file. Had added it as reminder. Pushed a commit with the right one